virt/release_notes/virt-4-16-release-notes.adoc

```diff
@@ -60,6 +60,9 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-16-installation-update"]
 === Installation and update
 
+//CNV-38590
+* After upgrading to {VirtProductName} {product-version}, data volumes that were previously removed through garbage collection might be recreated. This behavior is expected and the recreated data volumes should be ignored now that data volume garbage collection is disabled.
```

Comment:
Just wondering is this should be a known issue?

/remove-label peer-review-in-progress

/label peer-review-done

Reply:
Leaving as-is for now since it's not a fixable "problem" just a new but expected behavior someone might notice.